psscriptanalyzer/Common-Helpers.psm1

```
@@ -0,0 +1,58 @@
+$Global:HelperCache = @{}
+
+function Get-Configs {
+    param([string]$Path = "C:\app\config")
+    cd $Path
+    $files = ls *.json
+    return $files | % { cat $_.FullName | ConvertFrom-Json }
+}
+
+function Test-Endpoints {
```

PSScriptAnalyzer (1.25.0)

[info] The cmdlet 'Test-Endpoints' does not have a help comment.

(PSProvideCommentHelp)